packages/ffe-accordion-react/src/AccordionItem.js

```diff
@@ -11,77 +11,53 @@ const createClasses = (baseClass, isOpen, type) =>
     });
 
 class AccordionItem extends Component {
-    constructor(props) {
-        super();
-        this.state = {
-            isOpen: props.isOpen || false,
-        };
+    state = {
+        isOpen: this.props.isOpen || false,
+    };
 
-        this.onClick = this.onClick.bind(this);
-        this.onClickEnterAndSpace = this.onClickEnterAndSpace.bind(this);
+    getIsOpen() {
+        if (this.props.isOpen !== undefined) {
+            return this.props.isOpen;
+        }
+        return this.state.isOpen;
     }
 
-    onClick(e) {
+    onClick = e => {
```

Comment:
We're using this eye-breaking syntax now? 😢

Reply:
We've used it several other places. It lets us skip having a constructor in most cases, which is nice 👍

Reply:
But we miss out on the explicit `.bind` which is a nicer thing...

I guess since it's in the language now we'll have to try to get used to seeing it.